lib/filter.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+//////////////////////////////
+// contains, is, isNot, doesNotContain, exists, matches
+//
+// Include
+//////////////////////////////
+const addModule = function (results, pkg) {
+  try {
+    results[pkg.package.name] = require(pkg.package.name);
+  }
+  catch (e) {
+    if (!results.hasOwnProperty('__unrequireable')) {
+      results.__unrequireable = [];
+    }
+    results.__unrequireable.push(pkg.package.name);
+  }
+  return results;
+}
+
+module.exports = function ModuleFilter(packages, key) {
+  /**
+    * Contains
+  **/
+  this.containsSync = (search) => {
+    var results = {};
```

Comment:
We're not returning an array from this, we're returning a new object (notice the lack of `results` defined in your example, which is what actually needs to be returned).

Chaining is also irrelevant as an array isn't being returned.